### ISSUE TITLE
Choose a better default colormap and norm for HMISynopticMap

### ISSUE THREAD

**Issue by dstansby** [sunpy/sunpy#4890]
I just loaded a HMI synoptic map, and it looks like a big grey rectangle:
<img width="836" alt="Screenshot 2021-01-20 at 21 28 06" src="https://user-images.githubusercontent.com/6197628/105243026-62f5cd80-5b66-11eb-9792-97468710ce09.png">

We should choose a colormap and norm that make features in the map easier to spot.

**Comment by ayshih** [comment]
Well, it _should_ look like a big grey rectangle at this time period.  This time period is very quiet: the strongest feature is rather small, and will look even worse once you plot it shrunk down.  The active regions are too nascent to even be numbered yet; they do grow in strength and become numbered in the following day or two, but by that point, they've rotated away from being in this synoptic map.  Here's the corresponding image from SolarMonitor.org:
![shmi_maglc_fd_20181113_164632](https://user-images.githubusercontent.com/991759/105399119-363ac800-5bf1-11eb-99f6-a72c0b27c72f.png)
Simply put, this particular example is not a good baseline to judge generally applicable changes to colormap or normalization.

**Comment by dstansby** [comment]
I'm interested in the weaker magnetic fields, which are hard to distinguish with the default scaling. I played around a bit, and this seems to work well. In particular, it's now possible to distinguish the polarity of the poles, and old dispersed active region fields are also visible, without sacrificing the visibility of the stronger magnetic fields.
```python
hmi_map.peek(norm=mcolor.SymLogNorm(linthresh=20), cmap='RdBu')
```
![fig1](https://user-images.githubusercontent.com/6197628/105403491-b75d8380-5c20-11eb-8049-9221fb3d4657.png)

**Comment by ayshih** [comment]
I think your colormap/normalization looks really nice, but I'm not convinced that the _default_ settings should be changed.  It's not a standard presentation of HMI imagery (as far as I'm aware), and there will be scientists who are _not_ interested in the weaker magnetic fields.

This would certainly be a good gallery example, and perhaps even a convenience method.
